storage/src/vespa/storage/persistence/filestorage/modifiedbucketchecker.cpp

```diff
@@ -97,7 +97,9 @@ ModifiedBucketChecker::onClose()
     if (_singleThreadMode) {
         return;
     }
-    assert(_thread);
+    if (!_thread) {
+        return; // Aborted startup; onOpen() was never called so there's nothing to close.
```

Comment:
No need to notify _cond ?

Reply:
`_cond` is only waited on by the worker thread, which doesn't exist in this case.